src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -124,9 +126,9 @@ private async Task ExecuteAsync(HttpContext context)
             catch (Exception e)
             {
                 await _serializer.WriteErrorAsync(context, "An error occurred while processing the GraphQL query", e);
-                _logger ??= context.RequestServices.GetRequiredService<ILogger<GraphQLMiddleware>>();
-                _logger.LogError("An error occurred while processing the GraphQL query, {e}", e);
-                return;
+                _logger.LogError(e, "An error occurred while processing the GraphQL query.");
+
+                await Task.CompletedTask;
```

Comment:
This is not the same. This now causes the rest of the code below to execute, as opposed to the `return`. @hishamco why did you ask for this?

Reply:
Oops, I thought it was the end of the function, but after I saw the whole class I was wrong